.travis.yml

```diff
@@ -1,15 +1,23 @@
 language: java
 jdk:
 - oraclejdk8
-install: true
 env:
   global:
     - LOCAL_JMX=no
+    - TEST=""
+    - GRIM_MIN=1
+    - GRIM_MAX=1
   matrix:
-    - CASSANDRA_VERSION=2.1.16
-    - CASSANDRA_VERSION=2.2.8
-    - CASSANDRA_VERSION=3.0.9
-    - CASSANDRA_VERSION=3.10
+#    - CASSANDRA_VERSION=2.1.18
```

Comment:
@adejanovski i could not get 2.1.16 to work, not even without multiple reaper instances…
the JmxProxy could not connect to *any* of ccm nodes.
i have no idea why yet, but suspect it is just a my machine problem.
would be great if you could confirm this!